listenbrainz/mbid_mapping/config.py.sample

```diff
@@ -9,6 +9,8 @@ SQLALCHEMY_TIMESCALE_URI = "dbname=listenbrainz_ts user=listenbrainz_ts host=lis
 
 # If you are connecting to a musicbrainz-docker instance for MusicBrainz data, this string should work:
 MBID_MAPPING_DATABASE_URI = "dbname=musicbrainz_db user=musicbrainz host=localhost port=25432 password=musicbrainz"
+MB_DATABASE_MASTER_URI = "dbname=musicbrainz_db user=musicbrainz host=localhost port=25432 password=musicbrainz"
```

Comment:
I would like to see more comments here, documenting what these are and why someone might need to use them. Or, rather, that most people won't need to bother with this at all.

Reply:
Done